examples/basic_device.go

```diff
@@ -57,7 +56,10 @@ func ExecuteBasicDevice() {
 		os.Exit(1)
 	}
 	iface := interfaces.AstarteInterface{}
-	if err := json.Unmarshal(byteValue, &iface); err != nil {
+	// Parse the interface from a JSON string. Use interfaces.ParseInterface
```

Comment:
Change the comment - in general, using json.Unmarshal is deprecated by all means when loading interfaces and that was something we should have nuked a long time ago